### PR TITLE
Collapsible Navigation

### PR DESCRIPTION
**Pull request by nummi**
## Description
 - Removed "Advanced" title — _Not sure what purpose this really served_
 - Icon for app picker when collapsed — _Better than showing three letters. Unless those three letters are "App" ?_
 - New "toggle sidebar" icon — _Looks cleaner to my eyes. No more arrow but I think people will figure it out_
 - "Submit an Issue" text converted to icon — _Bug okay? It's not only for submitting bugs but I think communicates well enough_ 
 - Lightened icon color, darkened text

---

# ❓

  1. Looking for feedback on how the collapsed mode works. To avoid scrollbar hell I added an overflow menu. It's currently a native select.
  1. Also, should we save the toggle state to localstorage?

---

## Screenshots
expanded
collapsed

image

animation
_sorry for the terrible GIF_

dark

### CODE REVIEW

**Comment by rwwagner90** [comment]
This looks awesome @nummi! I need to pull this down and play with it, then will give some more feedback.

File: app/components/side-nav.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    return this.router.currentRouteName;
+  }
+
+  get overflowItemIsActive() {
+    return this.overflowItems.find((item) => {
+      return this.currentRouteName.match(item.route);
+    });
+  }
+
+  _createNavigationData() {
```
Comment (nummi) [inline app/components/side-nav.js:38]: Code Climate doesn't like this method. ¯\_(ツ)_/¯

**Comment by rwwagner90** [comment]
@nummi looks like we have a failing test 😩

**Comment by rwwagner90** [comment]
Since master is failing with the same issues, I am going to merge and debug.

**Pull request merged**